src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -236,11 +236,11 @@ Namespace Microsoft.CodeAnalysis.Semantics
             For Each base In statement.LocalDeclarations
                 If base.Kind = BoundKind.LocalDeclaration Then
                     Dim declaration = DirectCast(base, BoundLocalDeclaration)
-                    builder.Add(OperationFactory.CreateVariableDeclaration(declaration.LocalSymbol, Create(declaration.InitializerOpt), _semanticModel, declaration.Syntax))
+                    builder.Add(OperationFactory.CreateVariableDeclaration(Of EqualsValueSyntax)(declaration.LocalSymbol, Create(declaration.InitializerOpt), _semanticModel, declaration.Syntax))
```

Comment:
>EqualsValueSyntax [](start = 78, length = 17)

I think the check should be stronger, not just the type of the syntax, but also we should verify relationship between it and the syntax for ```statement```.